### PR TITLE
BUG Parameterize CV object with random seed

### PR DESCRIPTION
**Pull request by xdavio**
Addresses #23

### CODE REVIEW

**Comment by wlattner** [comment]
Thanks @xdavio, this looks good. Would you mind adding a test asserting we pass `random_state` to `KFold` and `StratifiedKFold `?

**Comment by xdavio** [comment]
@wlattner I've added the tests, give it another pass?

File: glmnet/util.py [hunk 1]
```diff
@@ -13,8 +13,9 @@
 from .scorer import check_scoring
 
 
-def _score_lambda_path(est, X, y, sample_weight, relative_penalties, cv, scoring, classifier,
-                       n_jobs, verbose):
+def _score_lambda_path(est, X, y, sample_weight, relative_penalties,
+                       n_splits, random_state, scoring, n_jobs,
+                       verbose):
```
Comment (wlattner) [inline glmnet/util.py]: I think you can now remove `n_splits` and `random_state` now that cv is handled by the estimator.

Comment (xdavio) [reply glmnet/util.py]: Fixed

**Comment by wlattner** [comment]
LGTM

**Pull request merged**